README.md

```diff
@@ -3,118 +3,67 @@
 [![][travis img]][travis]
 [![][license img]][license]
 
-# Knot.x - reactive template assembler
+#Knot.x - reactive multisource assembler 
+
 <p align="center">
   <img src="https://github.com/Cognifide/knotx/blob/master/icons/180x180.png?raw=true"
          alt="Knot.x"/>
 </p>
 
-Knot.x is a lightweight and high-performance **reactive template assembler**. It allows you to get rid of all the dynamic data from your content repository and put it into a fast and scalable world of microservices.
 
-![Flow diagram](icons/architecture/flow-diagram.png)
+## What is Knot.x?
+Very efficient, high-performance and scalable platform which assembles static and dynamic content from multiple sources.
+
+## What's philosophy behind Knot.x?
+
+### Stability and responsiveness
+**Knot.x** uses asynchronous programming principles which allows it to process a large number of requests using a single thread.
+Asynchronous programming is a style promoting the ability to write non-blocking code (no thread pools).
+The platform stays responsive under heavy and varying load and is designed to follow [Reactive Manifesto](http://www.reactivemanifesto.org/) principles.
+
+### Loose coupling
+Relies on asynchronous message-passing to establish a boundary between system components that ensures 
+loose coupling, isolation and location transparency. Base **Knot.x** component is called [Knot](https://github.com/Cognifide/knotx/wiki/Knot).
 
 We care a lot about speed and that is why we built it on [Vert.x](http://vertx.io/), known as one of the leading frameworks for performant, event-driven applications.
 
+### Scalability
+Various scaling options are available to suit client needs and help in cost optimization. Using a 
+simple concurrency model and message bus **Knot.x** can be scaled within a single host or cluster of 
+servers
+
+
+## What problems does Knot.x solve?
+**Knot.x** joins content from multiple sources on Repository and Data Ingestion levels to render dynamic pages:
```

Comment:
maybe?
"Knot.x assembles static and dynamic content to produce page with dynamic data in very performant manner."